app/controllers/admin/themes_controller.rb

```diff
@@ -261,6 +261,22 @@ def diff_local_changes
     end
   end
 
+  def update_single_setting
+    params.require("name")
+    @theme = Theme.find_by(id: params[:id])
+    raise Discourse::InvalidParameters.new(:id) unless @theme
+
+    setting_name = params[:name].to_sym
+    new_value = params[:value] || nil
+
+    previous_value = @theme.included_settings[setting_name]
+    @theme.update_setting(setting_name, new_value)
+    @theme.save
+
+    log_theme_setting_change(setting_name, previous_value, new_value)
+    render json: {}, status: :ok
```

Comment:
Could we return the serialized setting here? It would be more consistent with other RESTful routes.